vault/dendron.tutorial.md

```diff
@@ -2,28 +2,27 @@
 id: 678c77d9-ef2c-4537-97b5-64556d6337f1
 title: Getting Started
 desc: ''
-updated: 1631317031955
+updated: 1639190763313
 created: 1608051264282
 nav_order: 1.1
 ---
 
-
 ## Pre-requisites
 
 This walkthrough will help you get started with Dendron. Before you start, make sure you do the following:
 - installed [VSCode](https://code.visualstudio.com/download) or a VSCode compatible editor like [VSCodium](https://vscodium.com/)
-- installed [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron) from the code marketplace
+- installed [Dendron](https://link.dendron.so/vscode) from the code marketplace
 - initialize your first Dendron workspace
 
-When you first install Dendron, you can click on the green `Initialize Workspace` button on the Welcome screen to initialize your first workspace. Then, pick a location to store your notes, and you'll be brought to the tutorial experience, which will contain notes with the same tutorial content here.
+When you first install Dendron, you can click on the green `Get Started` button on the Welcome screen to initialize your first workspace. Then, pick a location to store your notes, and you'll be brought to the tutorial experience, which will contain notes with the same tutorial content here.
```

Comment:
for the tutorial, lets try to keep it terse and get to the point.

s/"you can click on the green..."/click on the `Get Started` button to begin/

Reply:
at this point, we haven't explained what a workspace is. lets spend a sentence explaining this

Reply:
the last part is outdated - you no longer pick a location to store your notes. you can cut